hikey.mk

```diff
@@ -123,26 +138,33 @@ edk2-clean: edk2-clean-common
 ################################################################################
 LINUX_DEFCONFIG_COMMON_ARCH ?= arm64
 LINUX_DEFCONFIG_COMMON_FILES ?= $(LINUX_PATH)/arch/arm64/configs/defconfig \
-				$(CURDIR)/kconfigs/hikey.conf
+				$(CURDIR)/kconfigs/hikey.conf \
+				$(PATCHES_PATH)/kernel_config/usb_net_dm9601.conf \
+				$(PATCHES_PATH)/kernel_config/ftrace.conf
```

Comment:
In hikey_optee, ftrace.conf is commented out but usb_net_dm9601.conf is actually included. I figured it's probably easier for the users to enable everything and have them use it if needed or ignore if not. Easier than changing makefile before building perhaps..